### PR TITLE
Enable RPM tests back

### PR DESCRIPTION
**Pull request by raphaelgavache**
Similarly to deb, removing installation steps from rpm scripts
Enabled back tests for Centos and Redhat

Follow-up
- lib injection tests

### CODE REVIEW

**Review by paullegranddc — APPROVED** [review]

**Review by BaptisteFoy — APPROVED** [review]

File: omnibus/package-scripts/installer-rpm/posttrans
Comment (chouquette) [inline omnibus/package-scripts/installer-rpm/posttrans:1]: Why not simply delete the whole file?

File: omnibus/package-scripts/installer-rpm/preinst
Comment (chouquette) [inline omnibus/package-scripts/installer-rpm/preinst:1]: same question here

**Review by chouquette — APPROVED** [review]
LGTM besides the empty files that could probably be removed

**Pull request merged**